Automattic/wp-calypso
# A4A > Agency Tier: Implement benefits sections

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1284

Design: GSGFOUvQQRQbi3rdxEesDB-fi-8577_9820

Note: The top section will be implemented in a different PR.

## Proposed Changes

This PR implements the benefits sections for agency tiers.

## Why are these changes being made?

* To showcase the agency tier benefits. 

## Testing Instructions

- Patch D163771-code if not merged already.
- Open the A4A live link
- Go to Agency Tier > Verify the screen looks good on all the different sizes as shown below.

![screencapture-agencies-localhost-3000-agency-tier-2024-10-15-12_22_05](https://github.com/user-attachments/assets/1f6214fa-dda5-49e1-b34d-28f391d8de91)

<img width="772" alt="Screenshot 2024-10-15 at 12 04 32 PM" src="https://github.com/user-attachments/assets/c3d50543-471e-4aaf-8bf9-6f4ef3cc5c3c">


## Pre-merge Checklist
